homework/insensitive-palindrom/palindrom.cpp

```diff
@@ -0,0 +1,14 @@
+#include <algorithm>
+#include <cctype>
+#include <iostream>
+
+bool is_palindrome(std::string palindrome)
+{
+    std::string removeSpecialSigns;
+    std::copy_if(palindrome.begin(), palindrome.end(), std::back_inserter(removeSpecialSigns), [](char ch){ return std::isalnum(ch);});
+    std::cout << removeSpecialSigns << '\n';
+    std::transform(removeSpecialSigns.begin(), removeSpecialSigns.end(), removeSpecialSigns.begin(), [](auto& ch){return std::tolower(ch);});
+    std::cout << removeSpecialSigns << '\n';
```

Comment:
Use transform and erse -> iterator returned by transform will be last valid, and rest of it you can erase. So you avoid copy_if